locales/en/index.yml

```diff
@@ -2013,6 +2018,12 @@ bonus:
           active: Active
           closed: Closed
           inactive: Upcoming
+      superCashback:
+        rankLabel: "{{position}}º user in rank"
+        rankTitle: "Super Cashback ranking"
+        howItWorks:
+          title: "How does it work?"
+          body: "The **100 thousand citizens** with the highest number of transactions **receive the Super Cashback, a prize of €1,500.**\n\n**The ranking changes every day.** The final one will be displayed on {{endDate}}."
```

Comment:
In this case we should use `bpdPeriod.minPosition` instead of the hardcoded text "100 thousand" (the min ranking could be different for different periods).
We should also use `bpdPeriod.superCashbackAmount`instead of the hardcoded text "€1.500" (the max cashback amount could be different for different periods).